pinot-common/src/main/proto/worker.proto

```diff
@@ -68,7 +68,13 @@ message StagePlan {
 message StageMetadata {
   repeated string instances = 1;
   repeated string dataSources = 2;
-  map<string, SegmentList> instanceToSegmentList = 3;
+  map<string, SegmentMap> instanceToSegmentMap = 3;
```

Comment:
we can add another map<string,SegmentList> right? one for real time and one for offline.. thought your approah is more flexible if we add more tabletype

Reply:
yes this way it is more flexible